src/components/VR/vr.js

```diff
@@ -4,7 +4,7 @@ const VR = () => {
 
     return (
         <div className='pt-10' style={{ textAlign: 'center', marginTop: '50px' }}>
```

Comment:
```suggestion
        <div className='pt-10'>
```

Reply:
That styling is for text that was removed, shouldn't be there.